### DISCUSSION TITLE
how to obtain a specific seed when you are generating images in a bunch?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [comfyanonymous/ComfyUI#1151] [Q&A]
Just what the title says, Even dragging and dropping the image in UI doesn't give you the seed. So what I am supposed to do here?

**Comment by stephantual** [comment]
same question

**Comment by ltdrdata** [comment]
If you loaded the image through drag and drop, the seed set in the KSampler on it is the seed directly associated with it.

If you are talking about batches, you can think of one seed as the seed for the entire latents.
There is no concept of the seed for the nth latent within a batch.

The seed is intended for reproduction under the assumption that the entire workflow has executed.

FYI,
The `KSampler (Inspire)` in the `Inspire Pack` has a feature that assigns a separate seed to each latent in the batch, allowing the apply noise individually for each latent.

**Comment by DrakenZA** [comment]
> Just what the title says, Even dragging and dropping the image in UI doesn't give you the seed. So what I am supposed to do here?

Due to the way ComfyUI handles generating noise, the only way to get lets say, image 4 in a batch, is to redo the whole batch again.

ComfyUI takes a single seed, and 'applies' it to the batch of latents, as a whole. It uses a single seed.

This means, the images you would get for a batch of 4, would not be the same you get for a batch of 3 or 2 or a single image using the seed.

That means the only way to re-produce x from a batch, is to redo the full batch.

It does not work like Auto, nor InvokeAI. Though as ltdrdata stated, if you use something l ike Inspire to override comfyui, then you would be able to easily re-produce those outputs without doing full batches.
